app/src/main/java/com/example/projektpo/MainActivity.kt

```diff
@@ -2,83 +2,87 @@ package com.example.projektpo
 
 import android.content.Intent
 import android.content.pm.ActivityInfo
-import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 import android.view.Gravity
 import android.view.ViewGroup
 import android.widget.Button
 import android.widget.LinearLayout
 import android.widget.ScrollView
-import java.util.*
+import androidx.appcompat.app.AppCompatActivity
 
 class MainActivity : AppCompatActivity() {
 
 
-
-
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
-        this.setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_PORTRAIT);
+        this.requestedOrientation = ActivityInfo.SCREEN_ORIENTATION_PORTRAIT
 
-
-        //////////////////buttonId/////////////
-        var buttonId=0
-        var buttonText="null"
         //////////////////////////////table with id///////////////////
-
-        var textYamlTable=arrayOf("bieganie","gotowanie","pływanie","umieranie","bieganie","gotowanie","pływanie","umieranie","bieganie","gotowanie","pływanie","umieranie","umieranie","bieganie","gotowanie","pływanie","umieranie")
+        val textYamlTable = arrayOf(
+            "bieganie",
+            "gotowanie",
+            "pływanie",
+            "umieranie",
+            "bieganie",
+            "gotowanie",
+            "pływanie",
+            "umieranie",
+            "bieganie",
+            "gotowanie",
+            "pływanie",
+            "umieranie",
+            "umieranie",
+            "bieganie",
+            "gotowanie",
+            "pływanie",
+            "umieranie"
+        )
 
         /////////////////////creating buttons///////////////////////////
         setContentView(R.layout.activity_main)
         title = "Projekt"
         val scrollView = ScrollView(this)
         val layoutParams = LinearLayout.LayoutParams(
             ViewGroup.LayoutParams.MATCH_PARENT,
-            ViewGroup.LayoutParams.MATCH_PARENT)
+            ViewGroup.LayoutParams.MATCH_PARENT
+        )
         scrollView.layoutParams = layoutParams
         val linearLayout = LinearLayout(this)
         val linearParams = LinearLayout.LayoutParams(
             ViewGroup.LayoutParams.MATCH_PARENT,
-            ViewGroup.LayoutParams.WRAP_CONTENT)
+            ViewGroup.LayoutParams.WRAP_CONTENT
+        )
         linearLayout.orientation = LinearLayout.VERTICAL
         linearLayout.layoutParams = linearParams
 
         scrollView.addView(linearLayout)
 
-        for (i in 0..textYamlTable.size-1) {
+        for ((i, value) in textYamlTable.withIndex()) {
```

Comment:
In `kotlin` range based `for` loop is not necessary, it is possible to use iterators for looping over content for more readable code.